### DISCUSSION TITLE
当在vite + vue3中配置了postcss-px-to-viewport  tailwind不生效问题

### DISCUSSION THREAD

**Discussion by zpaaaaa** [tailwindlabs/tailwindcss#12220] [Help]
我在vite + vue3中配置了postcss-px-to-viewport 其中将px转换为vw 发现tailwind的class不生效  一旦在vite.config.js中取消了当前postcss-px-to-viewport的配置就没问题了

**Comment by alex-krasikau** [comment]
Hello @zpaaaaa!

Could you please send your postcss configuration?
